pkgs/data/themes/catppuccin-plymouth/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, lib, fetchFromGitHub, variant ? "macchiato" }:
+
+assert lib.assertMsg
```

Comment:
Should we use `lib.checkListOfEnum`?

Reply:
I am not familiar with this, but the example seems to assume this kind of use case.
https://github.com/NixOS/nixpkgs/blob/628d4bb6e9f4f0c30cfd9b23d3c1cdcec9d3cb5c/lib/trivial.nix#L395-L405

Reply:
It does provide a nicer error message and it's slightly simpler this way, so I'll go with your suggestion.